bitnami/airflow/README.md

```diff
@@ -1287,6 +1288,16 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 23.0.0
+
+This major release adds support for Airflow 3 versions. Additionally, previous Airflow 2 images can be deployed by setting the corresponding image parameters. The chart logic will detect which image version you are using and it will generate the required Airflow configuration and Kubernetes objects.
```

Comment:
```suggestion
This major release adds support for Airflow `3.y.z` series. Additionally, previous Airflow `2.y.z` series can be deployed by setting the corresponding image parameters. The chart logic will detect which image version you are using and it will generate the required Airflow configuration and Kubernetes objects.
```